backend/frontend/tests/test_page_users_and_teams_nav.py

```diff
@@ -70,7 +70,54 @@ def test_load_more_button_appears(self):
         url = self.reverse_url("view_partial_teams_list")
         self.page.goto(url)
         self.page.wait_for_load_state()
-        
+
         body = self.page.locator("body")
 
         expect(body).to_contain_text("Load more")
+
+    def test_user_sees_teams_that_match_the_search_term(self):
```

Comment:
See if you can make the playwrite part of the testing as tight and simple as possible and then test the logic in a different way. Eg check that the frontend actually searches, then use normal unit tests to make sure that the search results are correct in different circumstances.

Reply:
Not sure how to check that the frontend searches without making the API calls. Does this mean I need to mock the calls?

Reply:
Nevermind, I removed the playwright test